packages/core/src/components/essentials/tooltip.ts

```diff
@@ -83,7 +92,9 @@ export class Tooltip extends Component {
 	addTooltipEventListener() {
 		// listen to move-tooltip Custom Events to move the tooltip
 		this.services.events.addEventListener(Events.Tooltip.MOVE, (e: CustomEvent) => {
-			if (this.tooltip.classed('hidden') === false) {
+			if (this.lastTriggeredEventType !== Events.Toolbar.SHOW_TOOLTIP &&
+				this.tooltip.classed('hidden') === false
+			) {
```

Comment:
I have to keep this `lastTriggeredEventType` to prevent a display issue when combining keyboard and mouse.

Can use following steps to verify on:
https://deploy-preview-1766--carbon-charts-angular.netlify.app/?path=/story/utility-truncations--truncated-labels-simple-bar

1. Put to mouse over truncated label title --> See the tooltip
2. Keep pressing `Tab` until one of the toolbar (e.g.: Show as table) got focus and show the tooltip.
3. Slightly move the mouse over truncated label title 
---> 
The tooltip is shown as "Show as table", but not the label title.